frog/imports/internalActivities/ac-wiki/config.js

```diff
@@ -0,0 +1,116 @@
+// @flow
+import * as React from 'react';
+import Form from 'react-jsonschema-form';
+import { FormControl } from 'react-bootstrap';
+import { listWikis, listPages } from '/imports/client/Wiki/helpers';
+
+const SelectWiki = ({ formContext, onChange, value = '' }: any) => (
```

Comment:
It would probably be possible to combine SelectWiki and SelectPage, but easier to leave it separate for now.